Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -91,7 +91,7 @@ if [ -z ${SCRIPT} ]; then
 	fi
 
 	while [ $n -lt $num_vehicles ]; do
-		spawn_model ${PX4_SIM_MODEL} $n
+		spawn_model ${VEHICLE_MODEL:=iris} $n
```

Comment:
Could you maybe set the defaults explictly above? this way we are more aware of the defaults. e.g. `model=${VEHICLE_MODEL:=iris}` above and take ${model} here

Reply:
Okay I've done
````
	m) VEHICLE_MODEL=${OPTARG:=iris};;
```
is that what you were looking for?